src/projects/admin.py

```diff
@@ -46,15 +46,15 @@ class ProjectAdmin(ImportExportActionModelAdmin):
     resource_class = ProjectResource
     formats = [base_formats.CSV, base_formats.XLSX]
     form = ProjectForm
-    list_display = ['name']
+    list_display = ['name', 'date_created', 'status']
```

Comment:
```suggestion
    list_display = ['name', 'status', 'date_created']
```
Habituellement on met la date à la toute fin 😁

Reply:
Ah cool je ne savais pas. :)